src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1394,32 +1394,20 @@ public void Add(Double summand)
         }
     }
 
-    public sealed class SdcaBinaryTrainer : SdcaTrainerBase<SdcaBinaryTrainer.Options, BinaryPredictionTransformer<TScalarPredictor>, TScalarPredictor>
+    /// <summary>
+    /// SDCA is a general training algorithm for (generalized) linear models such as support vector machine, linear regression, logistic regression,
+    /// and so on. SDCA binary classification trainer family includes several sealed members:
+    /// (1) <see cref="SdcaNonCalibratedBinaryTrainer"/> supports general loss functions and returns <see cref="LinearBinaryModelParameters"/>.
+    /// (2) <see cref="SdcaBinaryTrainer"/> essentially trains a regularized logistic regression model. Because logistic regression
+    /// naturally provide probability output, this generated model's type is <see cref="CalibratedModelParametersBase{TSubModel, TCalibrator}"/>.
+    /// where <see langword="TSubModel"/> is <see cref="LinearBinaryModelParameters"/> and <see langword="TCalibrator "/> is <see cref="PlattCalibrator"/>.
+    /// (3) <see cref="LegacySdcaBinaryTrainer"/> is used to support classical command line tools where model is weakly-typed to
```

Comment:
>LegacySdcaBinaryTrainer [](start = 23, length = 23)

Why you mention it if it's not publicly available?

Reply:
It's just a design doc. I will either move or delete it.

---
In reply to: [256655211](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256655211) [](ancestors = 256655211)